scripts/config.py

```diff
@@ -116,9 +132,11 @@ def gather_user_data_prompt():
     data['dns_secondary'] = check_ip('Secondary DNS', '8.8.4.4')
     data['local_cidr'] = check_cidr('VPN IPv4 local CIDR', '10.11.12.0/24')
 
-    duo_config = get_duo_data()
-    if duo_config:
-        data['duo_config'] = duo_config
+    mfa_type = get_mfa_type()
+    if mfa_type == 'duo':
+        data['duo_config'] = get_duo_data()
+    elif mfa_type == 'okta':
+        data['okta_config'] = get_okta_data()
 
     data['foxpass_api_key'] = prompt('Foxpass API Key')
```

Comment:
do you need to set data['mfa_type'] ?

Reply:
ah, yeah probably.

Reply:
double checked this. harmless no-op because we check it again later